2023-11-27 14:46:52,520:INFO:这是一条INFO级别的日志
2023-11-27 14:46:52,520:DEBUG:这是一条DEBUG级别的日志
2023-11-27 14:46:52,525:ERROR:
2023-11-27 14:46:52,525:ERROR:Processing issues:   0%|          | 0/1 [00:00<?, ?it/s]
2023-11-27 14:46:52,525:DEBUG:Processing mfem issue2878
2023-11-27 14:46:52,526:ERROR:
2023-11-27 14:46:52,526:ERROR:Processing test cases for mfem issue2878:   0%|          | 0/2 [00:00<?, ?it/s]
2023-11-27 14:46:52,526:ERROR:[A
2023-11-27 14:46:52,535:DEBUG:No existing branch named branch_d9275d7, continue...
2023-11-27 14:46:52,599:DEBUG:Switched to new branch branch_d9275d7 at d9275d734e060fc653236ab4901b370cbb6dd7ad
2023-11-27 14:46:52,600:DEBUG:Target directory: /root/mfem-code-analyzer/bugs/issue2878
2023-11-27 14:46:52,600:DEBUG:Copied tests/unit/fem/test_pa_coeff.cpp to /root/mfem-code-analyzer/bugs/issue2878/test_pa_coeff.cpp
2023-11-27 14:46:52,742:DEBUG:Deleted old branch branch_d9275d7
2023-11-27 14:46:52,836:DEBUG:Switched to new branch branch_d9275d7 at d9275d734e060fc653236ab4901b370cbb6dd7ad
2023-11-27 14:46:52,836:DEBUG:Target directory: /root/mfem-code-analyzer/bugs/issue2878
2023-11-27 14:46:52,836:DEBUG:Copied tests/unit/fem/test_pa_grad.cpp to /root/mfem-code-analyzer/bugs/issue2878/test_pa_grad.cpp
2023-11-27 14:46:52,836:ERROR:
2023-11-27 14:46:52,836:ERROR:Processing test cases for mfem issue2878: 100%|##########| 2/2 [00:00<00:00,  6.44it/s]
2023-11-27 14:46:52,836:ERROR:[A
2023-11-27 14:46:52,836:ERROR:[A
2023-11-27 14:46:52,836:INFO:Processing issue: {'name': 'issue2878', 'bug_sha': '78d65bd549cfd0aa2bcc2f9d91e0df1aa3f32538', 'fix_sha': 'd9275d734e060fc653236ab4901b370cbb6dd7ad', 'testcase': 'tests/unit/fem/test_pa_coeff.cpp,tests/unit/fem/test_pa_grad.cpp', 'testname': 'H1 pa_coeff,Hcurl/Hdiv pa_coeff,Hcurl/Hdiv mixed pa_coeff,ParallelPAGradient,PA Gradient,Nonlinear Convection', 'url': 'https://github.com/mfem/mfem/commit/987d2e043bad3a99841b88354dce0c0932cb8642'}
2023-11-27 14:46:52,836:INFO:Running bug version: 78d65bd549cfd0aa2bcc2f9d91e0df1aa3f32538
2023-11-27 14:46:52,979:DEBUG:No existing branch named branch_78d65bd, continue...
2023-11-27 14:46:53,057:DEBUG:Switched to new branch branch_78d65bd at 78d65bd549cfd0aa2bcc2f9d91e0df1aa3f32538
2023-11-27 14:46:53,057:ERROR:
2023-11-27 14:46:53,057:ERROR:Processing issues:   0%|          | 0/1 [00:00<?, ?it/s]
2023-11-27 14:46:53,057:ERROR:Traceback (most recent call last):
2023-11-27 14:46:53,057:ERROR:  File "/root/mfem-code-analyzer/get_normal_testcase_covarage/run_mfme_testcase.py", line 299, in <module>
2023-11-27 14:46:53,057:ERROR:processor.process_yaml_file()
2023-11-27 14:46:53,057:ERROR:  File "/root/mfem-code-analyzer/get_normal_testcase_covarage/run_mfme_testcase.py", line 147, in process_yaml_file
2023-11-27 14:46:53,057:ERROR:self.process_mfem_issue(issue_data)
2023-11-27 14:46:53,057:ERROR:  File "/root/mfem-code-analyzer/get_normal_testcase_covarage/run_mfme_testcase.py", line 102, in process_mfem_issue
2023-11-27 14:46:53,058:ERROR:self.copy_testcase(name, tc)
2023-11-27 14:46:53,058:ERROR:  File "/root/mfem-code-analyzer/get_normal_testcase_covarage/run_mfme_testcase.py", line 251, in copy_testcase
2023-11-27 14:46:53,058:ERROR:shutil.copy(source_path, target_path)
2023-11-27 14:46:53,058:ERROR:  File "/usr/lib/python3.10/shutil.py", line 417, in copy
2023-11-27 14:46:53,058:ERROR:copyfile(src, dst, follow_symlinks=follow_symlinks)
2023-11-27 14:46:53,058:ERROR:  File "/usr/lib/python3.10/shutil.py", line 256, in copyfile
2023-11-27 14:46:53,058:ERROR:with open(dst, 'wb') as fdst:
2023-11-27 14:46:53,058:ERROR:FileNotFoundError
2023-11-27 14:46:53,058:ERROR::
2023-11-27 14:46:53,058:ERROR:[Errno 2] No such file or directory: '/root/mfem/tests/unit/fem/test_pa_coeff.cpp'
